tests/test_interface.rb

```diff
@@ -1350,20 +1325,18 @@ def test_interface_ipv4_all_interfaces
     rescue Minitest::Assertion
       # clean up before failing
       config(*cfg)
-      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
+      interface_ethernet_default(interfaces[1])
```

Comment:
Looks like `default interface Gig0/0/0/0` deletes the interface, but `bundle id 10` doesn't stick around afterwards.

```bash
RP/0/RP0/CPU0:BigSunstone-for-Puppet-278-suns(config)#show running-config
[snip]
interface Loopback1
!
interface MgmtEth0/RP0/CPU0/0
 ipv4 address 192.168.1.7 255.255.255.0
!
interface GigabitEthernet0/0/0/0
 bundle id 10 mode on
!
interface GigabitEthernet0/0/0/1
 shutdown
!
interface GigabitEthernet0/0/0/2
 shutdown
!
! Configure default route.
router static
 address-family ipv4 unicast
  0.0.0.0/0 192.168.1.1
 !
!

RP/0/RP0/CPU0:BigSunstone-for-Puppet-278-suns(config)#default interface GigabitEthernet0/0/0/0
RP/0/RP0/CPU0:BigSunstone-for-Puppet-278-suns(config)#commit

RP/0/RP0/CPU0:BigSunstone-for-Puppet-278-suns(config)#show running-config
[snip]
interface Loopback1
!
interface MgmtEth0/RP0/CPU0/0
 ipv4 address 192.168.1.7 255.255.255.0
!
interface GigabitEthernet0/0/0/1
 shutdown
!
interface GigabitEthernet0/0/0/2
 shutdown
!
! Configure default route.
router static
 address-family ipv4 unicast
  0.0.0.0/0 192.168.1.1
 !
!
```